### PR TITLE
feat: allow NBT on wand items

### LINKED ISSUE #2383: NBT on items in configuration file
### What feature do you want to see added?

I want you to add CustomModelData support to the plugin. For example in the file worldedit-config.yml 
wand-item: 'PAPER {CustomModelData: 10162}'

### Are there any alternatives?

I do not know...

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by dordsor21**
- closes #2383

### CODE REVIEW

**Review by SirYwell — APPROVED** [review]

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/SelectionCommands.java [hunk 1]
```diff
@@ -325,22 +328,29 @@ public void wand(
...
+            parserContext.setSession(session);
+            try {
+                wand = WorldEdit.getInstance().getItemFactory().parseFromInput(wandId, parserContext);
+            } catch (InputParseException e) {
+                player.print(Caption.of("worldedit.wand.invalid"));
+                return;
+            }
         }
-        player.giveItem(new BaseItemStack(itemType, 1));
+        System.out.println("a "+ wand);
```
Comment (PierreSchwang) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/SelectionCommands.java:345]: ^

Comment (dordsor21) [reply worldedit-core/src/main/java/com/sk89q/worldedit/command/SelectionCommands.java:345]: Stop finding my debug

**Review by PierreSchwang — APPROVED** [review]
Untested, but looks good so far

**Comment by dordsor21** [comment]
> Untested, but looks good so far

I tested setting damage to an axe and it worked so I called it okay. No idea how the specific use in the issue would work but Minecraft has forever allowed for custom NBT on basically everything so there's no reason for NBT that is consequently picked up by something else to be set (and incidentally I tested this because of the toLowerCase on the config strings as damage is "Damage" in NBT :) )

**Review by NotMyFault — APPROVED** [review]

**Pull request merged**